include/deal.II/lac/arpack_solver.h

```diff
@@ -199,38 +225,59 @@ class ArpackSolver : public Subscriptor
   /**
    * Exceptions.
    */
-  DeclException2 (ExcInvalidNumberofEigenvalues, int, int,
+  DeclException2 (ArpackExcInvalidNumberofEigenvalues, int, int,
                   << "Number of wanted eigenvalues " << arg1
                   << " is larger that the size of the matrix " << arg2);
 
-  DeclException2 (ExcInvalidNumberofArnoldiVectors, int, int,
+  DeclException2 (ArpackExcInvalidEigenvectorSize, int, int,
+                  << "Number of wanted eigenvalues " << arg1
+                  << " is larger that the size of eigenvectors " << arg2);
+
+  DeclException2 (ArpackExcInvalidEigenvectorSizeNonsymmetric, int, int,
+                  << "To store all the real and some additional complex columns for " << arg1
+                  << "eigenvectors, its size equal to " << arg2
```

Comment:
I know this is formulated a bit confusing, all I want the user to let to know if he asked for 4 pairs and gives a length 4 that he should have provided 5 (i.e. n+1) instead.
"To store all the real and some additional complex columns for 4 eigenvectors, its size equal to 4 should be greater or equal to 5".
Suggestions for better formulations are very welcome.